### DISCUSSION TITLE
Online Demo?

### DISCUSSION THREAD

**Discussion by coolaj86** [marp-team/marp#136] [Q&A]
Is there an online demo of a presentation that's built with this somewhere?

Something like this?

- [Markdown Slides Demo](https://github.com/dadoomer/markdown-slides)
- [Markdown Slides Demo Source](https://raw.githubusercontent.com/dadoomer/markdown-slides/master/example/presentation.md)

The home page is stunning and the marketing is great, but I'm really just looking for a demo and example to cherry pick from, not learn the whole system out of the gate. Try before you buy and all that.

**Comment by yhatt** [comment]
I think [yhatt/marp-cli-example](https://github.com/yhatt/marp-cli-example) (and [source](https://github.com/yhatt/marp-cli-example/blob/master/PITCHME.md)) is the closest what you are looking for. [Check out the example section of this repository.](https://github.com/marp-team/marp#examples)

Unlike other presentation tools, Marp is actually focusing to the experience of creating presentation rather than playing. [Marp Web](https://web.marp.app) also may be good one if you want the demo like that. (NOTE: Marp Web is based on former Marp and no longer maintained)

**Reply by coolaj86** [reply]
Yes.

These are what I was looking for:
- Demos: https://yhatt.github.io/marp-cli-example/#1 | https://alexsci.com/test-marp-action/#1 | https://eyssette.github.io/teaching-theme-for-marp/
- Source: https://raw.githubusercontent.com/dadoomer/markdown-slides/master/example/presentation.md

However, those examples really don't show off much. I don't see any transitions or images or videos or code blocks.

[The docs look amazing](https://marpit.marp.app/markdown) and make it sound like this can do amazing things, and this has tons of github stars... but I wish I could see a _great_ example of how it's used.

I love the idea of having a live preview of the slides as I'm writing them - that's really cool.

But if you're not focusing on the presentation aspect, why not build something great around reveal.js with Live Reload and have both a great creation experience and a great presentation experience?

Also, I just (moments ago) found [reveal-md](https://github.com/webpro/reveal-md), which seems to be quite similar to the description I just gave - but with really bad SEO, no demo, and plain-jane docs with not so much as syntax highlighting.

**Reply by yhatt** [reply]
Yeah, we hope to collect more awesome and creative examples from community.

On the other hand, Marp team does not want causing any trobule by adding a example/usage without authors agreement. So we are depending on pull request from Marp users for adding useful examples.

FYI you can find usages in the wild from these:

- Find repositories from dependency graph of Marp tools. (e.g. https://github.com/marp-team/marp-cli/network/dependents: It has included big names such as Microsoft, Facebook, and TC39)
- Find GitHub repositories tagged by `marp`: https://github.com/topics/marp

> But if you're not focusing on the presentation aspect, why not build something great around reveal.js with Live Reload and have both a great creation experience and a great presentation experience?

There is a demo of Marpit framework and reveal.js: https://codesandbox.io/s/nw80vrxvpp

To boost integration with other slide frameworks, we had planned the future of Marpit framework since 2 years ago: https://github.com/marp-team/marpit/issues/194. However, this RFC is not given attention from users and not recieved any feedbacks. :cry:
